pypeit/specobj.py

```diff
@@ -167,6 +167,8 @@ class SpecObj(datamodel.DataContainer):
                  'DEC': dict(otype=float, descr='Declination (J2000) decimal degree'),
                  'MASKDEF_ID': dict(otype=(int, np.integer), descr='Slitmask definition ID'),
                  'MASKDEF_OBJNAME': dict(otype=str, descr='Name of the object from the slitmask definition'),
+                 'FORCE_EXTRACT': dict(otype=bool, descr='Boolean indicating if this is a forced extraction '
```

Comment:
We need to make sure all manually extracted objects receive a true here.

Reply:
Added another attribute for MANUAL_EXTRACT